### PR TITLE
User Groups, Copy and Naming Convention

### PR DESCRIPTION
**Pull request by timodwhit**
Added: 
- Copy Recipe
- User Groups Recipe

Updated: 
-  Naming conventions from drupal-developer to vampd-developer

### CODE REVIEW

**Comment by timodwhit** [comment]
This helps to resolve #1. Resolves the issue with the .gitconfig

File: README.md [hunk 1]
```diff
@@ -27,14 +27,36 @@ Attributes
...
 
 Adds phpmyadmin
 
+#### vampd-developer::copy
+
+Copies files from a src to a destination
+```
+"vampd_developer": {
+    "copy": {
+        "/vagrant/,gitconfig": ["/root/.gitconfig", "/home/vagrant/.gitconfig"]
```
Comment (arknoll) [inline README.md]: should this be a period? /vagrant/.gitconfig

Comment (timodwhit) [inline README.md]: Fixed.

File: recipes/default.rb [hunk 2]
```diff
@@ -17,22 +17,34 @@
...
+  include_recipe 'vampd-developer::xhprof.io'
 end
 
-unless node[:drupal_developer][:percona_toolkit].nil?
-  include_recipe 'drupal-developer::perconatoolkit'
+unless node[:vampd_developer][:percona_toolkit].nil?
+  include_recipe 'vampd-developer::perconatoolkit'
+end
+
+unless node[:vampd_developer][:percona_toolkit].nil?
```
Comment (arknoll) [inline recipes/default.rb]: duplicates prior 3 lines of code.

Comment (timodwhit) [inline recipes/default.rb]: I think we should get rid of this file. It actually cause issues

**Pull request merged**